sdk/messaging/azservicebus/internal/stress/stress-test-resources.bicep

```diff
@@ -5,6 +5,7 @@ param baseName string = resourceGroup().name
 param testApplicationOid string
 
 var apiVersion = '2017-04-01'
+//var location = 'southeastasia'
```

Comment:
Remove rather than comment

Reply:
It's commented so I don't have to keep looking it up. We're in this weird in-between state where we have a manually created set of nodes for southeastasia, so you have to manually target them. (I'll delete it when we finalize all of that).

Reply:
Another comment added.